lib/web_ui/lib/src/engine/text/paragraph.dart

```diff
@@ -1113,6 +1128,13 @@ void _applyParagraphStyleToElement({
     if (style._fontFamily != previousStyle._fontFamily) {
       cssStyle.fontFamily = quoteFontFamily(style._fontFamily);
     }
+    if (style._shadows != previousStyle._shadows) {
+      final List<ui.Shadow> shadows = style._shadows;
+      if (shadows.length > 1) {
+        throw UnsupportedError('Multiple shadows on text not supported');
+      }
+      cssStyle.textShadow = '';
```

Comment:
Should this be assigned to `_shadowListToCss(style._shadows)`?

Reply:
Done